templates/nginx/nginx.conf.template

```diff
@@ -143,7 +143,7 @@ http {
         proxy_set_header  Host            $host;
         proxy_set_header  X-Real-IP       $remote_addr;
         proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_set_header  Connection      "";
```

Comment:
🤔 What are your thoughts on just removing this line entirely? I think it'd be more helpful for us to configure this setting on our service's nginx configurations instead of hardcoding anything for it here

Reply:
Yeah actually that makes more sense